### ISSUE TITLE
pnpm tauri info exits with error code 101

### ISSUE THREAD

**Issue by linuxuser586** [tauri-apps/tauri#2509]
### **Describe the bug**
Running `pnpm tauri info` exits with code 101

### **To Reproduce**
1. `pnpm init`
2. `pnpm add -D @tauri-apps/cli`
2. `pnpm add @tauri-apps/api`
3. `tauri init`
4. `pnpm tauri info`
5. Observe error

### **Expected behavior**
Display the correct versions and exit with no error. Also note that the incorrect version for `@tauri-apps/cli` is reported when using pnpm before my proposed fix.

### **Screenshots**
```sh
pnpm tauri info
Downloading Rust CLI...
Download Complete
 app:spawn Running "/home/barry/experiment/tauri/demo-1/node_modules/.pnpm/@tauri-apps+cli@1.0.0-beta.9/node_modules/@tauri-apps/cli/bin/tauri-cli tauri info" +0ms


Operating System - Ubuntu, version 20.04 X64

Node.js environment
  Node.js - 14.17.4
  @tauri-apps/cli - 1.0.0-beta.7 (outdated, latest: 1.0.0-beta.9)
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("unexpected character 't' while parsing major version number")', src/info.rs:323:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  @tauri-apps/api - tauri-apps
 app:spawn Command "/home/barry/experiment/tauri/demo-1/node_modules/.pnpm/@tauri-apps+cli@1.0.0-beta.9/node_modules/@tauri-apps/cli/bin/tauri-cli" failed with exit code: 101 +0ms
```

### **Platform and Versions (required):**

```
Operating System - Ubuntu, version 20.04 X64

Node.js environment
  Node.js - 14.17.4
  @tauri-apps/cli - 1.0.0-beta.9
  @tauri-apps/api - 1.0.0-beta.6

Global packages
  npm - 7.20.3
  pnpm - 6.13.0
  yarn - 1.22.4

Rust environment
  rustc - 1.54.0
  cargo - 1.54.0

App directory structure
/src-tauri
/node_modules

App
  tauri.rs - 1.0.0-beta.7
  build-type - bundle
  CSP - default-src blob: data: filesystem: ws: wss: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
  distDir - src-tauri/tauri.conf.json
  devPath - http://localhost:3000
```

### **Additional context**
My suggested fix produces the above output

**Comment by yankeeinlondon** [comment]
I am still getting this problem; what can I do to work around this? I am using:

- `@tauri-apps/cli 1.0.0-beta.10`
- `@tauri-apps/api 1.0.0-beta.8`
- pnpm: `6.14.7`
- OS: `macOS`
- node: `14.17.6`

**Comment by yankeeinlondon** [comment]
Note: I reinstalled everything with **npm** as I thought the error had something to do with **pnpm** but I get the same error. That said ... **yarn** _does_ work.

**Comment by linuxuser586** [comment]
@ksnyde the fix is only in the next branch and has not been released yet.

**Comment by lucasfernog** [comment]
Closed by #2510
